src/datatype.ts

```diff
@@ -185,11 +186,35 @@ export class Datatype {
    *
    * @param length Length of the generated number. Defaults to `1`.
    *
+   * @see faker.datatype.hexadecimal()
+   *
    * @example
    * faker.datatype.hexaDecimal() // '0xb'
    * faker.datatype.hexaDecimal(10) // '0xaE13F044fb'
+   *
+   * @deprecated
    */
   hexaDecimal(length = 1): string {
+    deprecated({
+      deprecated: 'faker.datatype.hexaDecimal()',
+      proposed: 'faker.datatype.hexadecimal()',
+      since: 'v6.1.0',
```

Comment:
Actually this will be 6.1.2

```suggestion
      since: 'v6.1.2',
```